### ISSUE TITLE
How to create y in the data to predict edge edits?

### ISSUE THREAD

**Issue by LanceKnight** [rusty1s/pytorch_geometric#1974]
Hello, I'm trying to predict the edge edits. The notebook examples show how to get a graph classification, which has one y label per data, and show how to get a node classification, which has the same number of y label as nodes. But I'm not sure for predicting edge edits, how can I create y?

 Also not sure how to write the loss function? Since the predicted edits and ground-truth edits may have be different in sizes. E.g. ground-truth: there are 3 edits but I only predict there will be 2 edits.

**Comment by LanceKnight** [comment]
Well not exactly about GED. In my case, one graph is given as the input (instead of two graphs in the link), the graph edits are known as ground-truth. The project is about given reactant molecules as a graph, trying to predict the edge edits (once we know the edge edit, we know the chemical bond change, and thus we know the product of the reaction. So I somehow need to provide these edge edits as y to feed the model.
